src/controller/AutoCommissioner.cpp

```diff
@@ -729,6 +728,16 @@ CHIP_ERROR AutoCommissioner::CommissioningStepFinished(CHIP_ERROR err, Commissio
                 report.stageCompleted = CommissioningStage::kScanNetworks;
             }
         }
+
+        if (err != CHIP_NO_ERROR && IsSecondaryNetworkSupported() && !IsTriedSecondaryNetwork() &&
```

Comment:
OK, so maybe that boolean should be called `TryingSecondaryNetwork()`?

Reply:
Done.